libs/generic-view/ui/src/lib/interactive/tooltip/tooltip.tsx

```diff
@@ -59,30 +61,47 @@ export const Tooltip: BaseGenericComponent<
         let top = 0
         let left = 0
 
+        const adjustedOffset: TooltipOffsetType = { ...offset }
+        const { isFlippedVertically, isFlippedHorizontally } = getFlipStatus(
+          originalPlacement.current,
+          placment
```

Comment:
Should be `placement`

Reply:
https://github.com/mudita/mudita-center/pull/2110/commits/845c79eba96c8c57fab1cb71f7302addb71776b0